pipelinewise/fastsync/commons/tap_mongodb.py

```diff
@@ -395,32 +428,21 @@ def _export_collection(self, export_dir: str, collection_name) -> str:
         """
         LOGGER.info('Starting export of table "%s"', collection_name)
 
-        url = (
-            f'mongodb://{self.connection_config["user"]}:{self.connection_config["password"]}'
-            f'@{self.connection_config["host"]}:{self.connection_config["port"]}/'
-            f'{self.connection_config["database"]}?authSource={self.connection_config["auth_database"]}'
-            f'&readPreference=secondaryPreferred'
-        )
+        cmd = [
+            'mongodump',
+            '--uri',
+            f'"{self.connection_config["connection_string"]}"',
+            '--forceTableScan',
+            '--gzip',
+            '-c',
+            collection_name,
+            '-o',
+            export_dir,
+        ]
 
-        if self.connection_config.get('replica_set', None) is not None:
-            url += f'&replicaSet={self.connection_config["replica_set"]}'
-
-        if self.connection_config.get('ssl', None) is not None:
-            url += f'&ssl={self.connection_config["ssl"]}'
-
-        return_code = subprocess.call(
-            [
-                'mongodump',
-                '--uri',
-                f'"{url}"',
-                '--forceTableScan',
-                '--gzip',
-                '-c',
-                collection_name,
-                '-o',
-                export_dir,
-            ]
-        )
+        LOGGER.info("Running: %s", cmd)
```

Comment:
Can you remove this line? we cannot log the command because it contains password in clear.